packages/optimistic-oracle/src/proposer.js

```diff
@@ -89,6 +89,14 @@ class OptimisticOracleProposer {
           return Array.isArray(x) && x.every((value) => typeof value === "string");
         },
       },
+      settleAllRequests: {
+        // `otherAccountsToSettle`: array of other accounts whose requests should be settled automatically.
+        value: [],
```

Comment:
this seems a bit strange. if this is a bool then why is this default an array? does the comment on line 93 make sence here?

Reply:
Wow, nice catch. This is a mistake.